### DISCUSSION TITLE
Use Account for participating in polls

### DISCUSSION THREAD

**Discussion by mpldr** [lukevella/rallly#1071] [Ideas]
When responding to a poll, the account is essentially unused. The email has to be entered manually, the poll is not shown in the account as "responded to" and you can't invite users to participate in a poll. My suggestion would be to add these benefits to having an account:

- prefill data
  - email required → use account email
  - name required → use the one on the account
- adding a "history" (maybe in a separate tab) for easier editing of previously responded-to polls
  - old ones can be hidden/removed/archived once their last date has passed

Thank you for coming to my TED talk :)

**Comment by NikoHadouken** [comment]
duplicate of https://github.com/lukevella/rallly/discussions/1029
